packages/devtools-launchpad/bin/chrome-driver.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+const spawn = require('child_process').spawn;
+
+const chromeBinary = "/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome"
```

Comment:
This seems to be a OS X specific solution. Is there a way this can be done in `local.config` or overridden there?